src/vm/exceptionhandling.cpp

```diff
@@ -5839,6 +5839,16 @@ void CleanUpForSecondPass(Thread* pThread, bool fIsSO, LPVOID MemoryStackFpForFr
 
 #ifdef FEATURE_PAL
 
+typedef enum
+{
+    _URC_FATAL_PHASE1_ERROR = 3,
+} _Unwind_Reason_Code;
+typedef enum
+{
+} _Unwind_Action;
```

Comment:
@janvorli I'm a bit worry that enum size is not fixed and if there are only few small members in enum, its size can be 1 byte (see -fshort-enums), what's bad because its size is fixed in assembler. Should we specify fixed underlaying type for these two enums, or it's okay to assume at always at least 4 bytes?